PaxLabs/mynewt-core
# PKF-621: Enable/disable wakeup/sleep interrupt accordingly. The sleep…

… interrupt will never clear due to the chip implementation. This will mask other interrupt (e.g. wakeup)  since there's no edge change any more.

The driver can not be test by itself. A separated pull request is submitted for review.

Test case:
1. leave the device on the table. Wait for at least 5 seconds. (to let the chip enters sleep mode)
2. pick it up and shake it 3+ seconds  for battery level
3. put it back on the table and wait for 5 seconds
4. pick it up and shake it 5+ seconds for BLE pairing.
Repeat this procedure with various waiting time. (for instance, 10 mins later)

Advanced test:
1. have a unit with accelerometer interrupt pin lead available for oscilloscope.
2. connect the pin and monitor it when it sits on table without any movement. The pin should stays low.
3. Try to move or touch the device. Interrupt signal should be seen on the oscilloscope.
